packages/composite-checkout-wpcom/src/components/radio-button.js

```diff
@@ -99,6 +99,8 @@ const RadioButtonWrapper = styled.div`
 	:hover svg {
 		filter: grayscale( 0 );
 	}
+
+	${handleWrapperDisabled};
```

Comment:
I was going to say that this seemed like a fragile pattern, but it [seems that it's also an official pattern](https://emotion.sh/docs/composition), so 👍. (Another option might be to create two styled components like `RadioButtonWrapper` and `RadioButtonWrapperDisabled`, the latter of which wraps the former, but I'm actually not sure what happens if you nest styled components that way.)

Reply:
Yeah, I used WordPress/components [styled components](https://github.com/WordPress/gutenberg/blob/master/packages/components/src/card/styles/card-styles.js#L30) as a pattern. I'd be happy to change this to improve it.

Reply:
Oh, nice. Even better. I'd say we should just keep it, since it's not unusual.